worker/worker.py

```diff
@@ -166,6 +167,7 @@ def worker(worker_info, password, remote):
 
 def main():
   print("Worker started in " + os.getcwd() + " ...\n")
+  worker_dir = path.join(path.dirname(path.realpath(__file__)))
```

Comment:
@ppigazzini Is worker_dir different from os.getcwd() in the previous line?

Reply:
@tomtor I copied that code from games.py

Reply:
@tomtor difference between `os.getcwd()` and `path.dirname(path.realpath(__file__))`

https://stackoverflow.com/questions/5137497/find-current-directory-and-files-directory